src/main/java/Sprites.java

```diff
@@ -0,0 +1,112 @@
+import org.json.JSONObject;
+import org.json.JSONArray;
+import java.util.Map;
+import java.util.HashMap;
+
+/**
+ * @version 1.0
+ * @author B. Clint Hall
+ */
+public class Sprites {
+    private Map<String, JSONObject> spriteMap;
+    /**
+     * The root of the sb2 json object is a Stage object.
+     * This constructs a Sprites object from a Stage JSONObject.
+     * @param stage root of an Sb2 file
+     */
+    public Sprites(JSONObject stage) {
+        JSONArray stageChildren = stage.optJSONArray("children");
+        if (stageChildren == null) {
+            stageChildren = new JSONArray();
```

Comment:
So, why do you construct a new JSONArray if the stageChildren comes back as null. That just means that there are no children, right?